src/Tree/index.tsx

```diff
@@ -149,6 +150,10 @@ class Tree extends React.Component<TreeProps, TreeState> {
       d3zoom()
         .scaleExtent(zoomable ? [scaleExtent.min, scaleExtent.max] : [zoom, zoom])
         // TODO: break this out into a separate zoom handler fn, rather than inlining it.
+        .filter(() => {
+          if (interactiveNode) return event.target.classList.contains(this.svgInstanceRef) || event.target.classList.contains(this.gInstanceRef) || event.shiftKey;
```

Comment:
What is the reasoning for `event.shiftKey` in the last OR statement?

Reply:
i found that the the zoom behaviour gets interrupted when the mouse happens to hover over a node while zooming. this should allow the user to enforce zoom while holding the shift key.

Reply:
Ahh now I see, very nice 👍 

I will surface this in the documentation for the prop, since people won't be aware of the shift key functionality otherwise.